app/assets/javascripts/transactions.js

```diff
@@ -51,39 +44,50 @@ $(function () {
     $('#new_transaction').show();
   });
 
-  $(document).on('change', '#transfer_bank_account_id', function(e) {
-    if ($('#transfer_exchange_rate').size()) {
-      show_hide_exchange_rate();
-    }
+  $(document).on('change', '#transfer_amount', function(e) {
```

Comment:
@Gataniel combile it to `$(document).on('change', '#transfer_amount, #transfer_exchange_rate', function(e) {`